executor/chroot/chroot_executor.go

```diff
@@ -66,7 +82,7 @@ func (e *Executor) Run(f def.Formula, j def.Job, d string) def.JobResult {
 }
 
 // Execute a formula in a specified directory. MAY PANIC.
-func (e *Executor) Execute(f def.Formula, j def.Job, d string) def.JobResult {
+func (e *Executor) Execute(f def.Formula, j def.Job, d string, outS, errS io.WriteCloser) def.JobResult {
```

Comment:
Basically, I think both Job and our currently-used BasicJob implementation should get cooler. I postponed that larger change in favour of mucking the by-convention function sigs; goal is that they'll shrink again when that's all figured out.